### PR TITLE
Add Ukrainian translation

### PR DESCRIPTION
**Pull request by Hikari-no-Tenshi**

### CODE REVIEW

**Comment by tiann** [comment]
Thanks for your contribution, but please fix the compile error.

File: manager/app/src/main/res/values-uk/strings.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    <string name="module_failed_to_disable">Не вдалося вимкнути модуль: %s</string>
+    <string name="module_empty">Немає встановлених модулів</string>
+
+    <string name="module">Модулі</string>
+    <string name="uninstall">Видалити</string>
+    <string name="module_install">Встановити</string>
+    <string name="install">Встановити</string>
+    <string name="reboot">Перезавантажити</string>
+    <string name="settings">Налаштування</string>
+    <string name="reboot_userspace">М'яке перезавантаження</string>
```
Comment (tiann) [inline manager/app/src/main/res/values-uk/strings.xml:40]: /home/runner/work/KernelSU/KernelSU/manager/app/src/main/res/values-uk/strings.xml:52:4: Failed to flatten XML for resource 'reboot_userspace' with error: Invalid unicode escape sequence in string

"{str}"
/home/runner/work/KernelSU/KernelSU/manager/app/src/main/res/values-uk/strings.xml:52:4: string/reboot_userspace does not contain a valid string resource.
> Task :app:mergeReleaseResources

Comment (tiann) [reply manager/app/src/main/res/values-uk/strings.xml:40]: Maybe it should be: `М\'яке перезавантаження`?

**Review by tiann — CHANGES_REQUESTED** [review]

File: manager/app/src/main/res/values-uk/strings.xml (hunk 1 above)
Comment (Hikari-no-Tenshi) [reply manager/app/src/main/res/values-uk/strings.xml]: Fixed.

**Review by tiann — APPROVED** [review]

**Pull request merged**